test/services/user/registration.test.js

```diff
@@ -11,25 +11,35 @@ const registrationService = app.service('registration');
 const registrationPinService = app.service('registrationPins');
 const testObjects = require('../helpers/testObjects')(app);
 
-const patchSchool = (system, schoolId) => schoolModel.findOneAndUpdate({ _id: schoolId }, {
-	$push: {
-		systems: system._id,
-	},
-}, { new: true }).lean().exec();
+const patchSchool = (system, schoolId) =>
+	schoolModel
+		.findOneAndUpdate(
+			{ _id: schoolId },
+			{
+				$push: {
+					systems: system._id,
+				},
+			},
+			{ new: true }
+		)
+		.lean()
+		.exec();
 
-const createAccount = (system) => accountModel.create({
-	lasttriedFailedLogin: '1970-01-01T00:00:00.000+0000',
-	activated: false,
-	username: 'fritz',
-	password: '',
-	systemId: system._id,
-});
+const createAccount = (system) =>
+	accountModel.create({
+		lasttriedFailedLogin: '1970-01-01T00:00:00.000+0000',
+		activated: false,
+		username: 'fritz',
+		password: '',
+		systemId: system._id,
+	});
 
-const createPin = (pin = 6716, email) => registrationPinModel.create({
-	verified: false,
-	email: email || `${Date.now()}@test.de`,
-	pin,
-});
+const createPin = (pin = 6716, email) =>
```

Comment:
Codacy found an issue: ['createPin' is assigned a value but never used.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6045788)